config/manager/manager.yaml

```diff
@@ -8,21 +8,21 @@ metadata:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: manager
   namespace: system
   labels:
-    control-plane: controller-manager
+    control-plane: opendatahub-operator
```

Comment:
Would we want to take this opportunity to use https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ ?

Reply:
@astefanutti Are you expecting to delete exiting label `control-plane` and add common labels i.e. `app.kubernetes.io/name` , `app.kubernetes.io/part-of` ?

@VaishnaviHire @zdtsw what is your thought on it ?

Reply:
@zdtsw  I'll create a separate issue for adding common labels. Let's merge this PR if changes looks ok.

Reply:
@ajaypratap003 yes, ideally we would take the opportunity to add common labels.

As these labels are immutable, the less they are changed the better. Otherwise the deployment name will have to be changed again. It's not technically impossible, but it might be better to keep the iterations minimum.

Reply:
thats exactly my concern, by changing the label which can cause deployment failure. 
thb, i am unsure the change in this PR will work.
i will need to see how the test result is first.

Reply:
@zdtsw the work-around is to change the operator deployment name, so OLM creates a new one during upgrade (and deletes the old one), instead of updating the existing one, so the new labels can be introduced.

Reply:
Below is the Github issue for adding common labels.
https://github.com/opendatahub-io/opendatahub-operator/issues/754

Reply:
@astefanutti corect. i missed the line 1731 in that case it should be fine.

but how should we do with downstream? are we going to rename it as well? 
QE will need get involved if any test is based on the check on deployment and also we need to ensure monitoring will work after rename.

Reply:
> but how should we do with downstream? are we going to rename it as well?
QE will need get involved if any test is based on the check on deployment and also we need to ensure monitoring will work after rename.

In general observability is not tight to the deployment name. But you're right, it's possible QE has some automation that depends on that name. So any reference to it will have to be updated accordingly. That's another reason why I'd recommend the common labels be added as part of that fix, if that's what we want, to reduce the overhead.

Reply:
my personal opinion: if we want to do it then do it in one PR with all the changes. normally to split to several PR is to reduce risk but if eventually this is already changing the name of the deployment, then preferable to get all pieces in.  otherwise when deal with the "following" issue will need change deployment name again?

Reply:
Absolutely, that'd be my opinion as well.

Reply:
@asanzgom Can you please confirm about these label changes, will these changes impact the tests in downstream ?

Reply:
@ajaypratap003 we do have Test Cases that make use of label selector

```
ods-ci/ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot:184:        Run Keyword And Continue On Failure    FAIL    Timeout- ${output} pods found with the label selector ${label_selector} in ${namespace} namespace
ods-ci/ods_ci/tasks/Resources/RHODS_OLM/uninstall/codeflare_uninstall.resource:43:    Log  Checking pod with label selector: ${selector}
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:43:    [Documentation]    Deletes an openshift pod by label selector
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:48:    ...    No PODS present with Label '${label_selector}' in '${namespace}' namespace, Check the label selector and namespace provide is correct and try again
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:55:    [Documentation]    Check existence of an openshift pod by label selector
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:74:    [Documentation]    Get the POD name based on namespace and label selector
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:81:    [Documentation]    Get the Pod status based on namespace and label selector
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:89:    ...    namespace and label selector and return the
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:90:    ...    name of all the pod with matching label selector
ods-ci/ods_ci/tests/Resources/Page/OCPDashboard/Pods/Pods.robot:100:         FAIL    No POD found with the provided label selector in a given namespace '${namespace}'
```